### DISCUSSION TITLE
Creating a docker container with sqlite and map the sqlite db to a volume path

### DISCUSSION THREAD

**Discussion by the100rabh** [nocodb/nocodb#413] [Q&A]
I could not figure out the parameter to be used to configure the docker volume to sqlite db file path. Can we configure this to something like /data/sqlite instead of root folder so that we can use docker volume to persist this data

**Comment by the100rabh** [comment]
I guess that is /usr/app/data/ 

This info probably should be documented somewhere

**Comment by pranavxc** [comment]
You can mount volume at `/usr/app/data/` since `0.10.6` and in older version use `/usr/src/app`.

#351
